regions_www/tabs/manual_body.html

```diff
@@ -157,6 +157,18 @@ <h4><a name="faqs"></a>Additional FAQs</h4>
                 </table>
               </div>
             </div>
+            <p><a href="javascript:toggleElement(&#39;a4&#39;)">4. What if I'm interested in trips not people? And does PCT account for return trips?</a></p>
+             <div id="a4" style="display:none">
+             <p>The PCT presents results in terms of numbers of people travelling to work or school in different ways – e.g. the number of commuters cycling to work in different scenarios.  This is in line with the input data we are using from the Census (for commuting) and the National School Census (for school travel).</p>
+             <p>We appreciate some users will be more interested in numbers of trips than numbers of people. We recommend estimating numbers of trips affected as: <b>Change in number of people * mean cycling trips per cyclist per week * 52.2 weeks in the year</b></p>
+             <p>The change in the number of people (e.g. an increase in cyclists or a decrease in drivers) is given in the PCT outputs. We estimate mean number of cycling trips per cyclist per week from the National Travel Survey (2010-2016) as:<ul>
+<li>5.1 for commuters who say cycling is their usual mode of travelling to school</li>
+<li>2.3 for primary school children whose parents say cycling is their usual mode of travelling to school</li>
+<li>5.1 for secondary school children whose parents say cycling is their usual mode of travelling to school</li></ul></p>
```

Comment:
This `</p>` isn't valid.  It looks ok but `<p>` tags can't contain `<ul>` tags (see [HTML spec](https://html.spec.whatwg.org/dev/grouping-content.html#the-p-element)) so it should be:

```html
<p>
  the paragraph text
</p>
<ul>
  <li>Item 1</li>
  <li>Item 2</li>
</ul>
```

Reply:
fixed, thanks